reporter/src/main/kotlin/reporters/ReportTableModel.kt

```diff
@@ -108,6 +108,11 @@ data class ReportTableModel(
          */
         val id: Identifier,
 
+        /**
```

Comment:
Would be nice to add one finding to the test asset so that the HTML rendered by the test shows a link.

Reply:
Done.